### PR TITLE
use BillingMode enum to store BillingMode state instead of state

### PR DESCRIPTION
**Pull request by WangLiNaruto**
Fixed eclipse-xpanse/xpanse#1620
image
image
image
image
image
image

### CODE REVIEW

File: src/components/content/order/migrate/SelectDestination.tsx [hunk 1]
```diff
@@ -79,9 +74,9 @@ export const SelectDestination = ({
     selectAvailabilityZones: Record<string, string>;
     setSelectAvailabilityZones: Dispatch<SetStateAction<Record<string, string>>>;
     currentFlavor: string;
-    billingModes: DeployRequest.billingMode[] | undefined;
-    selectBillingMode: string;
-    setSelectBillingMode: Dispatch<SetStateAction<string>>;
+    billingModes: Billing.defaultBillingMode[] | undefined;
```
Comment (swaroopar) [inline src/components/content/order/migrate/SelectDestination.tsx:77]: we must not use the defaultBillingMode type here.  Must be just the Billing.BillingMode

File: src/components/content/order/migrate/SelectDestination.tsx [hunk 2]
```diff
@@ -79,9 +74,9 @@ export const SelectDestination = ({
...
+    selectBillingMode: Billing.defaultBillingMode;
```
Comment (swaroopar) [inline src/components/content/order/migrate/SelectDestination.tsx:78]: we must not use the defaultBillingMode type here. Must be just the Billing.BillingMode

File: src/components/content/order/migrate/SelectDestination.tsx [hunk 3]
```diff
@@ -79,9 +74,9 @@ export const SelectDestination = ({
...
+    setSelectBillingMode: Dispatch<SetStateAction<Billing.defaultBillingMode>>;
```
Comment (swaroopar) [inline src/components/content/order/migrate/SelectDestination.tsx:79]: we must not use the defaultBillingMode type here. Must be just the Billing.BillingMode

**Review by swaroopar — CHANGES_REQUESTED** [review]

**Comment by Alice1319** [comment]
LGTM

**Pull request merged**